src/quadexpr.jl

```diff
@@ -149,10 +149,10 @@ function addconstraint(m::Model, c::QuadConstraint)
     end
     return ConstraintRef{Model,QuadConstraint}(m,length(m.quadconstr))
 end
-addconstraint(m::Model, c::Array{QuadConstraint}) =
+addconstraint(m::Model, c::AbstractArray{QuadConstraint}) =
     error("Vectorized constraint added without elementwise comparisons. Try using one of (.<=,.>=,.==).")
 
-function addVectorizedConstraint(m::Model, v::Array{QuadConstraint})
+function addVectorizedConstraint(m::Model, v::AbstractArray{QuadConstraint})
```

Comment:
I wasn't able to construct a test case involving nonstandard arrays of `QuadConstraint` that didn't already pass or call this unexported function directly, so I changed it back to `Array`.